roles/manage-users/README.md

```diff
@@ -59,3 +59,6 @@ Run the test ```ansible-playbook -i inventory create_idm.yml```
 * Passwords are not set/changed in this role
 * Modify IdM credentials in create_idm.yml to match your IdM
 * You can test modfications by re-running the test pointing to a different json file ```ansible-playbook -i inventory create_idm.yml -e "@vars/idm_mod.json"```
+
+## Output
+This role augments the `users` data structure `generate_password` for use with `manage-user-passwd` role
```

Comment:
Please provide the example structure here + drop the comment about "generate_password" and "manage-user-passwd" ... and just state something like: "This role augments the `users` data structure passed in and the final result is expected to be: 
```
....
...
... example here
... 
....
```